src/Tests/Helix.targets

```diff
@@ -0,0 +1,17 @@
+<Project>
+  <PropertyGroup Condition="'$(TargetFramework)' != ''">
+    <HelixDotnetTestArgs>$(TargetName) $(Configuration) $(TargetFramework) $(Platform) $(TestRunnerFilterArguments)</HelixDotnetTestArgs>
+    <HelixDotnetTestCommand Condition="'$(IsHelixPosixShell)' != 'true'">call %HELIX_CORRELATION_PAYLOAD%\dotnettest.cmd $(HelixDotnetTestArgs)</HelixDotnetTestCommand>
```

Comment:
Instead of having a bash and cmd script that do the same thing, let's try to use a single script type that is available on all targets e.g. python.

Reply:
Will consider this separately.